dash_auth/oauth.py

```diff
@@ -13,6 +13,8 @@ class OAuthBase(Auth):
     AUTH_COOKIE_NAME = 'dash_token'
     # Name of the cookie containing the OAuth2 access token
     TOKEN_COOKIE_NAME = 'oauth_token'
+    USERNAME_COOKIE = 'dash_user'
```

Comment:
Why not combine these into one cookie? Having fewer cookies is more efficient from a network point of view and easier to manage if the user needs to manually remove cookies.

Reply:
The user_data is not set by default so it's not there if you don't use it, the username get set by PlotlyAuth.